src/components/IOBanner.js

```diff
@@ -1,10 +1,26 @@
 import React from 'react';
 import { css } from '@emotion/react';
 import PropTypes from 'prop-types';
-import { Button, Link } from '@newrelic/gatsby-theme-newrelic';
+import {
+  Button,
+  Link,
+  useTessen,
+  useInstrumentedHandler,
+} from '@newrelic/gatsby-theme-newrelic';
 import ioBanner from '../images/ioBanner.png';
 
 const IOBanner = ({ isMobile }) => {
+  const tessen = useTessen();
+
+  const handleBannerButtonClick = useInstrumentedHandler(
+    () => {
+      tessen.track('instantObservability', 'BannerButtonClick', {});
+    },
+    {
+      actionName: 'BannerButtonClick',
+    }
+  );
```

Comment:
i don't think you need to use the `useInstrumentedHandler` hook here. i think you can just use tessen inside an inline function in the `onClick`

Reply:
How do we decide when this used / not used?

Reply:
After talking to @roadlittledawn I removed it.